app/views/static_pages/contributing.html.erb

```diff
@@ -1,16 +1,34 @@
 <%= title('Contributing') %>
 
-<div class="container tmp-container-spacer">
-  <div class="page-intro">
-    <h1 class="page-heading-title">How to Contribute</h1>
-    <p class="page-intro__text">
-      The Odin Project is an Open Source project, built and maintained by volunteers who dedicate their time and skills to making The Odin Project one of the best free education platforms on the web. We are always working on projects to improve Odin and are always looking for people who want to join our growing team of maintainers.</a>
-    </p>
-  </div>
+<div class='page-container'>
+  <div class="grid grid-cols-12">
+    <div class="col-span-12 md:col-start-3 md:col-span-8">
+      <section class='text-center pb-12'>
+        <h1 class='page-heading-title'>How to Contribute</h1>
+        <p>
+          The Odin Project is an Open Source project, built and maintained by volunteers who dedicate their time
+          and skills to making The Odin Project one of the best free education platforms on the web. We are always
+          working on projects to improve Odin and are always looking for people who want to join our growing team
+          of maintainers.
+        </p>
+      </section>
 
-  <%= render 'static_pages/contributing/contributing_options' %>
+      <%= render 'static_pages/contributing/contributing_options' %>
+    </div>
+  </div>
 </div>
 
-<%= render 'static_pages/contributing/benefits' %>
+<section class="bg-gray-100 odin-dark-bg-accent">
+  <div class="container">
+    <div class="grid grid-cols-12">
+      <div class="col-span-12 md:col-start-3 md:col-span-8">
+        <div class="page-container">
+          <h2 class="text-gold font-bold text-center pb-16 mb-0 text-2xl">Why you should get involved</h2>
+          <%= render(AboutPage::BeliefComponent.with_collection(t("contributing_page_beliefs"))) %>
```

Comment:
So I know this looks a little funky at the moment, but wanted to get some feedback before going all in on it. @leosoaivan 's component had almost the exact same layout as the one on this page, so I thought it would be a good spot for some re-use. 

If we're happy with this the next step would probably be to give it a more generic name, like `ImageCardComponent` or something similar. 

Only disadvantage of this is previously it was a 2x2 grid, now it's always stacked.

Reply:
Great spot with these components being the same 💪 

100% agree this is a great place for reuse. This looks like a media object to me:
https://tailwindui.com/components/application-ui/layout/media-objects
https://getbootstrap.com/docs/4.0/layout/media-object/

Perhaps that's the kind of naming we could use for the new component? -`MediaComponent`

> Only disadvantage of this is previously it was a 2x2 grid, now it's always stacked.
No worries with this, I think it's actually much easier to read when stacked in one column in that second screenshot.

Reply:
In each of the icons, one of the path elements will have a fill attribute with a value of `#F3F3F3`
Changing those values to `none` will get rid of the white backgrounds.

<img width="1213" alt="Screenshot 2022-10-02 at 12 16 00" src="https://user-images.githubusercontent.com/7963776/193451287-4a3d8479-1d93-45e0-87bc-04ab6301d09f.png">

The clock looks a little bit odd, but we can always come back and tweak these later.

Reply:
Ah, nice one. I did attempt to change some fill values but obviously not that one :sweat_smile: